frontend/packages/dev-console/src/components/dropdown/ResourceDropdown.tsx

```diff
@@ -72,10 +73,19 @@ class ResourceDropdown extends React.Component<ResourceDropdownProps, State> {
       this.setState({ title: <LoadingInline /> });
       return;
     }
+
     if (!this.props.loaded || !selectedKey) {
       this.setState({
         title: <span className="btn-dropdown__item--placeholder">{placeholder}</span>,
       });
+    } else {
+      const dropdownItems = _.cloneDeep(this.state.items);
```

Comment:
Why are you cloning the whole set of items?

Reply:
Yeah right, could have done without it.